docs/guides/mssqlserver/concepts/mssqlserver.md

```diff
@@ -0,0 +1,477 @@
+        subject:
+          organizationalUnits:
+            - server
+          organizations:
+            - kubedb
+        emailAddresses:
+          - dev@appscode.com
+      - alias: client
+        secretName: mssqlserver-client-cert
+        subject:
+          organizationalUnits:
+            - client
+          organizations:
+            - kubedb
+        emailAddresses:
+          - abc@appscode.com
+    clientTLS: true
+    issuerRef:
+      apiGroup: cert-manager.io
+      kind: Issuer
+      name: mssqlserver-ca-issuer
+  healthChecker:
+    periodSeconds: 15
+    timeoutSeconds: 10
+    failureThreshold: 2
+    disableWriteCheck: false
+  monitor:
+    agent: prometheus.io/operator
+    prometheus:
+      serviceMonitor:
+        labels:
+          release: prometheus
+        interval: 10s
+  version: 2022-cu12
+  deletionPolicy: Halt
+```
+
+### spec.version
+
+`spec.version` is a required field that specifies the name of the [MSSQLServerVersion](/docs/guides/mssqlserver/concepts/catalog.md) crd where the docker images are specified. Currently, when you install KubeDB, it creates the following `MSSQLServerVersion` resources,
+
+```bash
+$ kubectl get msversion
+NAME        VERSION   DB_IMAGE                                                DEPRECATED   AGE
+2022-cu12   2022      mcr.microsoft.com/mssql/server:2022-CU12-ubuntu-22.04                2d
+2022-cu14   2022      mcr.microsoft.com/mssql/server:2022-CU14-ubuntu-22.04                2d
+```
+### spec.replicas
+
+`spec.replicas` specifies the total number of primary and secondary nodes in SQL Server Availability Group cluster configuration. One pod is selected as Primary and others act as secondary replicas. KubeDB uses `PodDisruptionBudget` to ensure that majority of the replicas are available during [voluntary disruptions](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#voluntary-and-involuntary-disruptions).
+
+To learn more about how to setup a SQL Server Availability Group cluster (HA configuration) in KubeDB, please visit [here](/docs/guides/mssqlserver/clustering/ag_cluster.md).
+
+### spec.leaderElection
+
+There are three fields under MSSQLServer CRD's `spec.leaderElection`. These values define how fast the leader election can happen.
+
+- `leaseDurationSeconds`: This is the duration in seconds that non-leader candidates will wait to force acquire leadership. This is measured against time of last observed ack. Default 15 sec.
+- `renewDeadlineSeconds`: This is the duration in seconds that the acting master will retry refreshing leadership before giving up. Normally, LeaseDuration \* 2 / 3. Default 10 sec.
+- `retryPeriodSeconds`: This is the duration in seconds the LeaderElector clients should wait between tries of actions. Normally, LeaseDuration / 3. Default 2 sec.
+
+If the Cluster machine is powerful, user can reduce the times. But, Do not make it so little, in that case MSSQLServer will restart very often.
```

Comment:
Remove this. As raft rely on network, so you can suggest user to increase `period` and `electionTick` if they have high network latency's.